docs/widgets/baseNodes/element.md

```diff
@@ -0,0 +1,201 @@
+#### Element (from Node)
```

Comment:
This would be a follow up commit.

I can put it in the pull request if you like in fact I think would be good so they are a little better to navigate ( although the gitbook has an always present navbar on the left. /shrug